flight/Libraries/sanitycheck.c

```diff
@@ -136,6 +134,7 @@ int32_t configuration_check()
 			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_HORIZON:
 			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_AXISLOCK:
 			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_VIRTUALBAR:
+			case MANUALCONTROLSETTINGS_FLIGHTMODEPOSITION_FLIPOVERREV:
```

Comment:
Biggest set of changes is in sanity check, allowing the mode, allowing manual under specified circumstances on multirotor, allowing arming with the mode, etc.